payments/trustly/seamless-view.md

```diff
@@ -0,0 +1,298 @@
+---
+title: Swedbank Pay Trustly Payments
+sidebar:
+  navigation:
+  - title: Trustly Payments
+    items:
+    - url: /payments/trustly
+      title: Introduction
+    - url: /payments/trustly/redirect
+      title: Redirect
+    - url: /payments/trustly/seamless-view
+      title: Seamless View
+    - url: /payments/trustly/after-payment
+      title: After Payment
+    - url: /payments/trustly/other-features
+      title: Other Features
+---
+
+{% include jumbotron.html body="The Seamless View scenario represents the
+                                opportunity to implement Trustly directly in
+                                your webshop." %}
+
+## Introduction
+
+Seamless View provides an integration of the payment process directly on your
+website. This solution offers a smooth shopping experience with Swedbank Pay
+payment pages seamlessly integrated in an `iframe` on your website. The costumer
+does not need to leave your webpage, since we are handling the payment in the
+`iframe` on your page.
+
+![screenshot of the Trustly payment window][trustly-payment-embedded-view]{:height="425px" width="700px"}
+
+{% include alert-callback-url.md api_resource="trustly" %}
+
+## Step 1: Create the payment
+
+A Trustly payment is a straightforward way to perform a direct-bank payment. It is followed up by posting a capture, cancellation or reversal
+transaction.
+
+An example of an abbreviated `POST` request is provided below. Each individual field of the JSON document is described in the following section.
+An example of an expanded `POST` request is available in the
+[other features section].
+
+When properly set up in your merchant/webshop site and the payer starts the
+payment process, you need to make a POST request towards Swedbank Pay with your
+payment information. This will generate a payment object with a unique
+`paymentID`. You will receive a **JavaScript source** in response.
```

Comment:
Please write the name of the operation in which the JavaScript can be found. `view-payment`, is it?

```suggestion
`id` URI. You will receive a **JavaScript source** in response.
```